mailpoet/assets/js/src/automation/integrations/mailpoet/steps/add_tags/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { __ } from '@wordpress/i18n';
+import { tag } from '@wordpress/icons';
+import { StepType } from '../../../../editor/store';
+import { LockedBadge } from '../../../../../common/premium_modal/locked_badge';
+import { PremiumModalForStepEdit } from '../../../../../common/premium_modal';
+
+export const step: StepType = {
+  key: 'mailpoet:add-tag',
+  group: 'actions',
+  title: __('Add tag', 'mailpoet'),
+  description: __('Add a tag or multiple tags to a subscriber.', 'mailpoet'),
+  subtitle: () => <LockedBadge text={__('Premium', 'mailpoet')} />,
+  foreground: '#00A32A',
+  background: '#EDFAEF',
+  icon: () => (
+    <div style={{ width: '100%', height: '100%', scale: '1.4' }}>{tag}</div>
+  ),
+  edit: () => (
+    <PremiumModalForStepEdit
+      tracking={{
+        utm_medium: 'upsell_modal',
+        utm_campaign: 'create_automation_editor_add_tag',
+      }}
+    >
+      {' '}
+      {__('Adding tags is a premium feature.', 'mailpoet')}{' '}
```

Comment:
Is the trailing space necessary?

Reply:
[fec2ade](https://github.com/mailpoet/mailpoet/pull/4604/commits/fec2ade3a5b4c34a61f829a0e189baebe0cd976e) :facepalm:

Reply:
Hah, I meant the one at the end! There's one more! 😄 Probably not doing any harm...

Reply:
... [89b6675](https://github.com/mailpoet/mailpoet/pull/4604/commits/89b667543e8ac4b27f56be0b32328c85ad2c8be9)